internal/cmd/schema-registry/command_config_describe.go

```diff
@@ -26,58 +26,59 @@ type configOut struct {
 	RulesetOverrides   string `human:"Ruleset Overrides,omitempty" serialized:"ruleset_overrides,omitempty"`
 }
 
-func (c *command) newConfigDescribeCommand() *cobra.Command {
+func (c *command) newConfigDescribeCommand(cfg *v1.Config) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "describe",
-		Short: "Describe top-level or subject-level schema compatibility.",
+		Short: "Describe top-level or subject-level schema configuration.",
 		Args:  cobra.NoArgs,
 		RunE:  c.configDescribe,
-		Example: examples.BuildExampleString(
-			examples.Example{
-				Text: `Describe the configuration of subject "payments".`,
-				Code: "confluent schema-registry config describe --subject payments",
-			},
-			examples.Example{
-				Text: "Describe the top-level configuration.",
-				Code: "confluent schema-registry config describe",
-			},
-		),
 	}
 
+	example1 := examples.Example{
+		Text: `Describe the configuration of subject "payments".`,
+		Code: "confluent schema-registry config describe --subject payments",
+	}
+	example2 := examples.Example{
+		Text: "Describe the top-level configuration.",
+		Code: "confluent schema-registry config describe",
+	}
+	if !cfg.IsCloudLogin() {
+		example1.Code += " " + OnPremAuthenticationMsg
+		example2.Code += " " + OnPremAuthenticationMsg
+	}
+	cmd.Example = examples.BuildExampleString(example1, example2)
+
 	cmd.Flags().String("subject", "", SubjectUsage)
-	pcmd.AddApiKeyFlag(cmd, c.AuthenticatedCLICommand)
```

Comment:
It looks like the api key flags disappeared for this command instead of being marked hidden.